### PR TITLE
Update getting started files

### PR DESCRIPTION
**Pull request by Rachael-Graham**
Update sample helm install files after testing for 2.1

### CODE REVIEW

File: gloo-gateway/helm-install/2.1/agent.yaml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+# Global settings for agent configuration
+# Number of access logs to buffer per Envoy proxy
+accessLogsBufferSize: 50
+# Name of the workload cluster by which cluster is referenced in all Gloo Mesh configurations
+cluster: $REMOTE_CLUSTER
+# Set the logger to development mode, which can cause panics. Do not use in production.
+devMode: false
+# Install external authentication service in the gloo-mesh namespace
+ext-auth-service:
+  enabled: true
```
Comment (Rachael-Graham) [inline gloo-gateway/helm-install/2.1/agent.yaml:17]: left a [comment](https://github.com/solo-io/gloo-mesh-enterprise/pull/4954/files#diff-5f951b09dc7697d492523a817a98d2fc86807ecd353a2d9b160350a1326de7f6R218) to determine whether we should enable in gloo-mesh in gateway

File: gloo-gateway/getting-started/2.1/gs-single.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  cluster: $CLUSTER_NAME
+# Install the Gloo agent alongside the management server,
+# such as to run the management cluster also as a workload cluster in a single-cluster setup
+registerMgmtPlane:
+  enabled: true
+  # Configuration for managed Istio control plane and gateway installations by using the Istio Lifecycle Manager
+  managedInstallations:
+    enabled: true
+    images:
+      # Solo.io Istio distribution repository; required for Gloo Mesh Istio.
```
Comment (Nadine2016) [inline gloo-gateway/getting-started/2.1/gs-single.yaml:12]:
```suggestion
      # Solo.io Istio distribution repository; required for Gloo Istio.
```

File: gloo-gateway/getting-started/2.1/gs-single.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      # You get the repo key from your Solo Account Representative.
+      hub: $REPO
+      # The Solo.io Gloo Mesh Istio version
```
Comment (Nadine2016) [inline gloo-gateway/getting-started/2.1/gs-single.yaml:15]:
```suggestion
      # The Solo.io Gloo Istio version
```

File: gloo-gateway/helm-install/2.1/agent.yaml [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+# This is an example Helm values file for PRODUCTION-LEVEL installations.          #
+# You MUST edit some settings in this file to provide your own details.            #
+# To see the default settings instead, run                                         #
+# helm show values gloo-mesh-agent/gloo-mesh-agent --version $GLOO_GATEWAY_VERSION #
+####################################################################################
+
+# Global settings for agent configuration
+# Number of access logs to buffer per Envoy proxy
+accessLogsBufferSize: 50
+# Name of the workload cluster by which cluster is referenced in all Gloo Mesh configurations
```
Comment (Nadine2016) [inline gloo-gateway/helm-install/2.1/agent.yaml:11]:
```suggestion
# Name of the workload cluster by which cluster is referenced in all Gloo configurations
```

File: gloo-gateway/helm-install/2.1/agent.yaml [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.namespace
+  # Required for OpenShift installations: Allow the pod to be assigned a dynamic user ID.
+  floatingUserId: false
+  image:
+    pullPolicy: IfNotPresent
+    registry: gcr.io/gloo-mesh
+    repository: gloo-mesh-agent
+    # Gloo Mesh Enterprise patch version. Must match the gloo-mesh-mgmt-server version.
```
Comment (Nadine2016) [inline gloo-gateway/helm-install/2.1/agent.yaml:94]:
```suggestion
    # Gloo Gateway patch version. Must match the gloo-mesh-mgmt-server version.
```

File: gloo-gateway/helm-install/2.1/mgmt-server.yaml [hunk 6]
```diff
@@ -0,0 +1,325 @@
...
+        key: key
+        name: gloo-mesh-enterprise-license
+        optional: true
+  # Required for OpenShift installations: Allow the pod to be assigned a dynamic user ID.
+  floatingUserId: false
+  image:
+    pullPolicy: IfNotPresent
+    registry: gcr.io/gloo-mesh
+    repository: gloo-mesh-mgmt-server
+    # Gloo Mesh Enterprise patch version. For a list of patch versions,
```
Comment (Nadine2016) [inline gloo-gateway/helm-install/2.1/mgmt-server.yaml:132]:
```suggestion
    # Gloo Gateway patch version. For a list of patch versions,
```

File: gloo-gateway/helm-install/2.1/mgmt-server.yaml [hunk 7]
```diff
@@ -0,0 +1,325 @@
...
+  serviceType: ClusterIP
+  # Configuration for the console and envoy containers
+  sidecars:
+    console:
+      env: null
+      image:
+        pullPolicy: IfNotPresent
+        registry: gcr.io/gloo-mesh
+        repository: gloo-mesh-ui
+        # Gloo Mesh Enterprise patch version
```
Comment (Nadine2016) [inline gloo-gateway/helm-install/2.1/mgmt-server.yaml:250]:
```suggestion
        # Gloo Gateway patch version
```

File: gloo-gateway/helm-install/2.1/mgmt-server.yaml [hunk 8]
```diff
@@ -0,0 +1,325 @@
...
+          memory: 256Mi
+    envoy:
+      env:
+      - name: ENVOY_UID
+        value: "0"
+      image:
+        pullPolicy: IfNotPresent
+        registry: gcr.io/gloo-mesh
+        repository: gloo-mesh-envoy
+        # Gloo Mesh Enterprise patch version
```
Comment (Nadine2016) [inline gloo-gateway/helm-install/2.1/mgmt-server.yaml:266]:
```suggestion
        # Gloo Gateway patch version
```

File: gloo-mesh/getting-started/2.1/gs-single.yaml [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+  cluster: $CLUSTER_NAME
+# Install the Gloo agent alongside the management server,
+# such as to run the management cluster also as a workload cluster in a single-cluster setup
+registerMgmtPlane:
+  enabled: true
+  # Configuration for managed Istio control plane and gateway installations by using the Istio Lifecycle Manager
+  managedInstallations:
+    cluster: $CLUSTER_NAME
+    images:
+      # Solo.io Istio distribution repository; required for Gloo Mesh Istio.
```
Comment (Nadine2016) [inline gloo-mesh/getting-started/2.1/gs-single.yaml:12]:
```suggestion
      # Solo.io Istio distribution repository; required for Gloo Mesh Istio.
```
```suggestion
      # Solo.io Istio distribution repository; required for Gloo Istio.
```

File: gloo-mesh/getting-started/2.1/gs-single.yaml [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+      # You get the repo key from your Solo Account Representative.
+      hub: $REPO
+      # The Solo.io Gloo Mesh Istio version
```
Comment (Nadine2016) [inline gloo-mesh/getting-started/2.1/gs-single.yaml:15]:
```suggestion
      # The Solo.io Gloo Istio version
```

File: gloo-gateway/helm-install/2.1/agent.yaml [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+  rootTlsSecret:
+    name: relay-root-tls-secret
+  # Address and port by which gloo-mesh-mgmt-server in the Gloo management plane is accessed
+  serverAddress: $MGMT_SERVER_NETWORKING_ADDRESS
+  # Default self-signed certs: Reference to a secret containing a shared token for authenticating to the relay server
+  #tokenSecret:
+  #  key: token
+  #  name: relay-identity-token-secret
+  #  namespace: gloo-mesh
+# Enable eBPF sidecar acceleration to reduce network latency in your service mesh. Do not use in production.
```
Comment (Nadine2016) [inline gloo-gateway/helm-install/2.1/agent.yaml:61]: does this apply to gloo gateway?

File: gloo-gateway/helm-install/2.1/single-cluster.yaml [hunk 12]
```diff
@@ -128,11 +128,11 @@ glooMeshMgmtServer:
     repository: gloo-mesh-mgmt-server
     # Gloo Mesh Enterprise patch version. For a list of patch versions,
     # see https://docs.solo.io/gloo-mesh-enterprise/main/reference/changelog/
-    tag: $GLOO_MESH_VERSION
+    tag: $GLOO_GATEWAY_VERSION
```
Comment (Nadine2016) [inline gloo-gateway/helm-install/2.1/single-cluster.yaml:131]: line 129 myst be Gloo Gateway patch version

**Review by Nadine2016 — APPROVED** [review]

**Pull request merged**